Tutorials/Editors-Manual/Getting-Started-With-Umbraco/Adding-Tabs/index.md

```diff
@@ -0,0 +1,79 @@
+---
+versionFrom: 8.17.0
+versionTo: 9.0.0
```

Comment:
Since the article is valid for both the releases versionTo metadata will act as in the attached image 😀:
![image](https://user-images.githubusercontent.com/82437098/134907318-45d7f7bf-1541-4482-8c64-aa228a0d0974.png)
.

Reply:
Oh, if it's required to display it like that, just ignore me on this one 😇